### PR TITLE
stream switcher

### PR DESCRIPTION
**Pull request by kimlarocca**
pulled out stuff we don't need, refactored a bit so it's easier to read - let me know if the markup looks good to you

### CODE REVIEW

File: src/components/StreamSwitcherCard.vue [hunk 1]
```diff
@@ -81,162 +59,107 @@ export default {
...
+    left: 30px;
+    border-left: 20px solid transparent;
+    border-right: 20px solid transparent;
+    border-top: 15px solid RGB(var(--color-text-inverse));
+    opacity: 0;
+    -webkit-transition: opacity var(--animation-duration-slow) var(--animation-easing-incoming) 50ms;
+    -moz-transition: opacity var(--animation-duration-slow) var(--animation-easing-incoming) 50ms;
+    -ms-transition: opacity var(--animation-duration-slow) var(--animation-easing-incoming) 50ms;
+    -o-transition: opacity var(--animation-duration-slow) var(--animation-easing-incoming) 50ms;
+    transition: opacity var(--animation-duration-slow) var(--animation-easing-incoming) 50ms;
```
Comment (walsh9) [inline src/components/StreamSwitcherCard.vue:101]: it would be nice to get postcss and autoprefixer set up sometime

**Review by walsh9 — APPROVED** [review]
Looks good

File: src/components/StreamSwitcherCard.vue (hunk 1 above)
Comment (kimlarocca) [reply src/components/StreamSwitcherCard.vue:101]: Autoprefixing is included in Nuxt.js and is handled by autoprefixer - maybe I can get rid of this (I needed it for the Ember version)

Comment (kimlarocca) [reply src/components/StreamSwitcherCard.vue:101]: Also need to add a knob or something so people can toggle the active state on and off to preview the animation. I have an animation ticket and I will figure it out while working on that!

**Pull request merged**